benchmark/torch/ddpg/README.md

```diff
@@ -0,0 +1,27 @@
+## Reproduce DDPG with PARL
+Based on PARL, the DDPG algorithm of deep reinforcement learning has been reproduced, reaching the same level of indicators as the paper in Mujoco benchmarks.
+
+> Paper: DDPG in [Continuous control with deep reinforcement learning](https://arxiv.org/abs/1509.02971)
+
+### Mujoco games introduction
+Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
+
+### Benchmark result
+
+<img src=".benchmark/DDPG_results.png" width = "800" height ="400" alt="DDPG_results"/>
+
+## How to use
+### Dependencies:
++ python3.5+
++ [parl](https://github.com/PaddlePaddle/PARL)
++ gym
++ torch
++ mujoco-py>=1.50.1.0
```

Comment:
Please specify the version for the mujoco-py library using `==`.

Reply:
done